### PR TITLE
[CSLD-101] Docs for Bash scripts, README.

### PR DESCRIPTION
**Pull request by denisshevchenko**
1. This is only `README.md` for scripts, I didn't add `--help` description in the scripts yet.
2. `DeployCluster.hs` was removed because @domenkozar said that we don't need it anymore.

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+## Generate
+
+* `generate/certificates.sh` - generate certificates using [`postvend-app`](https://github.com/input-output-hk/postvend-app). Please make sure you have `postvend-cli` command in your `PATH`.
+* `generate/genesis.sh` - generate keys using `cardano-keygen`.
+
+## Haskell
+
+* `haskell/lint.sh` - `hlint` command for Cardano SL source code. It uses `HLint.hs`-settings (from the project's root).
+* `haskell/stylish.sh` - `stylish-haskell` command for Cardano SL source code. It uses `HLint.hs` as well.
```
Comment (gromakovsky) [inline scripts/README.md]: No, it doesn't use it, it explicitly ignores it because we don't want to format `HLint.hs`.

File: scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+* `haskell/update-cabal-versions.sh` - update Cardano SL version in all `.cabal`-files.
+* `haskell/recover-from-stack-clean.sh` - useful if you're using Atom editor with `haskell-ghc-mod`.
+
+## CI
+
+Please note that these scripts are for Travis CI only. These scripts rely on specific environment variables, so manual running of these scripts on your machine don't assumed.
```
Comment (gromakovsky) [inline scripts/README.md]: Not only for Travis, I see `appveyor-retry.cmd`.

File: scripts/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+## Haskell
+
+* `haskell/lint.sh` - `hlint` command for Cardano SL source code. It uses `HLint.hs`-settings (from the project's root).
+* `haskell/stylish.sh` - `stylish-haskell` command for Cardano SL source code.
+* `haskell/update-cabal-versions.sh` - update Cardano SL version in all `.cabal`-files.
+* `haskell/recover-from-stack-clean.sh` - useful if you're using Atom editor with `haskell-ghc-mod`.
+
+## CI
+
+Please note that these scripts are for Travis CI only. These scripts rely on specific environment variables, so manual running of these scripts on your machine is not implied.
```
Comment (gromakovsky) [inline scripts/README.md]: But wait, this comment: `Not only for Travis, I see appveyor-retry.cmd.` still applies, doesn't it?

Comment (denisshevchenko) [inline scripts/README.md]: Ops..

**Comment by gromakovsky** [comment]
Oh, wait. These changes should be in `cardano-sl-0.5-rc` because frontenders use it. Can you cherry-pick on top of that branch and change the base branch of this PR?

**Comment by denisshevchenko** [comment]
> These changes should be in cardano-sl-0.5-rc

Well, AFAIU, these changes should be in `master` too, right?

**Comment by denisshevchenko** [comment]
https://github.com/input-output-hk/cardano-sl/pull/999 <- this is for `cardano-sl-0.5-rc`.

**Comment by denisshevchenko** [comment]
Since https://github.com/input-output-hk/cardano-sl/pull/999 merged, this PR isn't required anymore (changes from `rc`-branch will be in `master` anyway).

**Pull request closed without merging**